pkg/storegateway/gateway_test.go

```diff
@@ -57,6 +57,13 @@ import (
 	"github.com/grafana/mimir/pkg/util/validation"
 )
 
+const (
+	// RingNumTokensDefault is the number of tokens registered in the ring by each store-gateway
+	// instance. We use a safe default instead of exposing to config option to the user
+	// in order to simplify the config.
+	RingNumTokensDefault = 512
+)
```

Comment:
i was thinking of keeping the constant in gateway_ring.go so we can also use it instead of the hardcoded value of `512` in the flag declaration. In other words, just rename the previous `RingNumTokens` to `RingNumTokensDefault`. Perhaps we also don't need it exported - so `ringNumTokensDefault`

Reply:
ohh so you mean not to define the constant in the test file itself but define it in gateway.go file and make it private?

Reply:
exactly

Reply:
perfect doing that now.

Reply:
@dimitarvdimitrov addressed your comments :)